FRINXio/frinx-frontend
# Fd 540 network topology weight

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-540` 
| Tests Added + Pass?      | No
| Any Dependency Changes?  | No


HOW TO TEST:
thsi needs to be testet against running topology-discovery server - see https://github.com/FRINXio/frinx-inventory-server/pull/340
